sql_metadb/derived_tables/runlist.txt

```diff
@@ -6,6 +6,8 @@ agreements_subscription_agreement_org_ext.sql
 finance_transaction_invoices.sql
 finance_transaction_purchase_order.sql
 holdings_ext.sql
+holdings_notes.sql
+holdings_electronic_access
```

Comment:
`.sql `suffix missing

Reply:
HI @natalyapik the suffix is still missing.

Reply:
Oh I'm sorry, if I have been misleading. I just meant the suffix in the [runlist.txt file](https://github.com/natalyapik/folio-analytics/blob/hold_notes_eaccess/sql_metadb/derived_tables/runlist.txt) at line 10. In should be `holdings_electronic_access.sql` not `holdings_electronic_access`

Reply:
I just went to your branch Natalya. If you go to your hold_note_eaccess/sql_metadb/derived_tables/runlist.txt, you'll see in that list that your table name is missing the ".sql" at the end.